Cargo.toml

```diff
@@ -41,6 +41,9 @@ vecrem = { git = "https://github.com/WaffleLapkin/vecrem", rev = "6b9b6f42342df8
 [features]
 default = []
```

Comment:
You've probably forgotten to add either of `rustls`/`native-tls` to default

Reply:
As no backward compatibility requirements exist, I think this should be left as user choice.

Reply:
Or alternatively, we can set `rustls` as default. (Because it should just work on all targets)

Reply:
I think most users don't care about a TLS implementation and it'd be unpleasant to force them to solve nonsence tasks. `native-tls` is more widely used than `rustls`, so I think that it must be the default.